lib/ruby_libstorj/env.rb

```diff
@@ -13,8 +13,12 @@ class Env
     end
 
     def initialize(*options)
-      @pointer = ::LibStorj::Ext::Storj.method(:init_env).call(*options)
-      @pointer[:loop] = ::Libuv::Ext.default_loop
+      @storj_env = ::LibStorj::Ext::Storj.method(:init_env).call(*options)
+      @storj_env[:loop] = ::Libuv::Ext.default_loop
```

Comment:
I get a segfault if i comment out that line... i think libuv is loaded twice and they're different default loops...